src/logging.cc

```diff
@@ -1314,12 +1322,24 @@ void LogCleaner::Disable() {
   enabled_ = false;
 }
 
+void LogCleaner::UpdateCleanUpTime() {
+  const int64 next = (FLAGS_logcleansecs
+                      * static_cast<int64>(1000000));  // in usec
```

Comment:
The literal can be represented as `int` and will be automatically promoted to a 64-bit within the expression. The cast is therefore not necessary.

Reply:
You're right, I'll remove the cast.